modules/oxide/src/less/theme/components/menubar/menubar.less

```diff
@@ -11,9 +11,9 @@
 
 @menubar-background-color: @background-color;
 @menubar-background: url("data:image/svg+xml;charset=utf8,%3Csvg height='@{_menubar-height}' viewBox='0 0 40 @{_menubar-height}' width='40' xmlns='http://www.w3.org/2000/svg'%3E%3Crect x='0' y='@{_menubar-divider-height}' width='100' height='1' fill='@{_menubar-row-separator-color}'/%3E%3C/svg%3E") left 0 top 0 @menubar-background-color;
```

Comment:
Why do we have this if we're just making it transparent/not using it now? This seems like it would be better set to a specific color and then overridden to use the background image in the TinyMCE 5 skins.

Reply:
On that note... I don't see a Jira/task to fix the premium skins to account for these changes either? Has that been done, as when I tried to test it I get an error:
```
'src/less/theme/globals/global-variables.less' wasn't found. Tried - node_modules/@tinymce/oxide/src/less/skins/content/default/src/less/theme/globals/global-variables.less,npm://src/less/theme/globals/global-variables.less,src/less/theme/globals/global-variables.less in file node_modules/@tinymce/oxide/src/less/skins/content/default/content.less line no. 7
```